docs/manual/developer/07_understanding_build_system.md

```diff
@@ -62,9 +62,9 @@ CMake requires projects have an entry point called `/CMakeLists.txt`. This
 uses the CMake language and drives building and installing the project. This
 file contains several things:
 
- 1. The many build-time options for customizing the types of content generated,
- 2. The hand-off for generating each product's specific content,
- 3. Common installation, testing, and distribution targets.
+- The many build-time options for customizing the types of content generated,
```

Comment:
Was it your intent to convert this to an unordered list? If not, you replace `-` with `1.`. Markdown will auto-number them.